dbt_copilot_helper/commands/application.py

```diff
@@ -0,0 +1,264 @@
+            )
+
+            click.secho(
+                f"{'Task ID:':<10}{task}",
+                fg="green",
+            )
+            heading = f"{'Container Name':<35}{'CPU':<10}{'Memory':<10}{'Revision':<12}"
+
+            # Optional parameters.
+            if storage:
+                heading += f"{'Dsk Read':<12}{'Dsk Write':<12}"
+            if network:
+                heading += f"{'Net Read':<12}{'Net Write':<12}"
+            heading += "Image"
+
+            click.secho(
+                heading,
+                fg="cyan",
+            )
+
+        # Print container stats
+        result = f"{cont_name:<35}" + f"{cpu:<10}" + f"{memory:<10}" + f"{task_def_revision:<12}"
+
+        # Optional stats.
+        if storage:
+            result += f"{storage_read:<12}{storage_write:<12}"
+        if network:
+            result += f"{network_read:<12}{network_write:<12}"
+        result += f"{image}"
+
+        click.secho(
+            result,
+            fg="yellow",
+        )
+        index = index + 1
+
+
+@application.command()
+@click.option("--env", type=str, required=True)
+@click.option("--app", type=str, required=True)
+@click.option("--project-profile", type=str, required=True)
+@click.option("--disk", is_flag=True)
+@click.option("--storage", is_flag=True)
+@click.option("--network", is_flag=True)
+def stats(env, app, project_profile, disk, storage, network):
+    """Deploy image tag to a service, defaults to image tagged latest."""
+
+    # Query string to get the required container stats
+    query_string = "stats max(CpuUtilized), max(MemoryUtilized), max(EphemeralStorageUtilized) by TaskId, TaskDefinitionFamily, TaskDefinitionRevision, StorageReadBytes, StorageWriteBytes, NetworkRxPackets, NetworkTxBytes, KnownStatus | filter Type='Task' | sort TaskId desc"
+    cpu_response = get_query_results(env, app, project_profile, query_string)
+
+    click.echo(
+        click.style(
+            f"\n{'Name:':<20}",
+            fg="green",
+        )
+        + click.style(f"{app}", fg="green", bold=True)
+    )
+
+    click.echo(
+        click.style(
+            f"{'No of instances:':<20}",
+            fg="green",
+        )
+        + click.style(len(cpu_response["results"]), fg="green", bold=True)
+    )
+
+    # Add to title if additional parameters are used.
+    heading = f"\n{'Type':<10}{'TaskID':<35}{'Revision':<10}{'Status':<12}{'CPU':<10}{'Memory':<10}"
+    if disk:
+        heading += f"{'Disk':<10}"
+    if storage:
+        heading += f"{'Dsk Read':<12}{'Dsk Write':<12}"
+    if network:
+        heading += f"{'Net Read':<12}{'Net Write':<12}"
+
+    click.secho(
+        heading,
+        fg="cyan",
+    )
+    for (
+        task,
+        taskdef,
+        task_def_revision,
+        storage_read,
+        storage_write,
+        network_read,
+        network_write,
+        status,
+        cpu,
+        memory,
+        dsk,
+    ) in cpu_response["results"]:
+        result = (
```

Comment:
Wondering if the output might look better in a table, something like [this](https://github.com/uktrade/copilot-tools/blob/main/dbt_copilot_helper/commands/config.py#L155-L187).

Reply:
updated.